Sources/StreamChatUI/Utils/ChatMessage+Extensions.swift

```diff
@@ -27,13 +27,17 @@ extension _ChatMessage {
     }
 
     /// Says whether the message is part of message thread.
-    var isPartOfThread: Bool {
+    var isThreadParent: Bool {
         let isThreadStart = replyCount > 0
         let isThreadReplyInChannel = showReplyInChannel
 
         return isThreadStart || isThreadReplyInChannel
     }
 
+    var isThreadChild: Bool {
+        parentMessageId != nil
+    }
+
```

Comment:
I'd suggest a little bit different naming:
- `isRootOfThread` -> the root message of the thread
- `isPartOfThread` -> all messages that are a part of thread

(btw - this extension should be public and needs documentation)

Reply:
✅